tests/system/components/authHeader.js

```diff
@@ -106,4 +107,35 @@ module.exports = {
         .assert.attributeContains('section > div > div > div > div > div:nth-child(4) > ul > li:nth-child(2) > a', "href", "help")
         .end();
     }
+
+    'Verify Account Nav can Be toggeled Open and closed': function(browser) {
+        browser
+        .click(selectors.headerProfileBtn)
+        .waitForAnimation()
+        .verify.elementsVisible(
+            selectors.headerAccountNav,
```

Comment:
comma isn't necessary here you can just use
`.verify.elementsVisible(selectors.headerAccountNav)`